CHANGELOG.yml

```diff
@@ -35,6 +35,23 @@ items:
     notes:
       - type: change
         title: Lock connection to namespace
+        body: >-
+          The behavior changed so that a connected Telepresence client is bound to a namespace. The namespace can then
+          not be changed unless the client disconnects and reconnects. A connection is also given a name. The default
+          name is composed from <code>&lt;kube context name&gt;-&lt;namespace&gt;</code> but can be given explicitly
+          when connecting using <code>--name</code>. The connection can optionally be identified using the option
+          <code>--use &lt;name match&gt;</code> (only needed when docker is used and more than one connection is active).
+      - type: change
+        title: Deprecation of global --context and --docker flags.
+        body: >-
+          The global flags <code>--context</code> and <code>--docker</code> will now be considered deprecated unless used
+          with commands that accept the full set of Kubernetes flags (e.g. <code>telepresence connect</code>).
+      - type: change
+        title: Deprecation of the --namespace flag for the intercept command.
+        body: >-
+          The <code>--namespace</code> flag is now deprecated for <code>telepresence intercept</code> command. The flag can instead
+          be used with all commands that accept the full set of Kubernetes flags (e.g. <code>telepresence connect</code>).
+      - type: change
         body: >-
           The behavior changed so that a connected Telepresence client is bound to a namespace. The namespace can then
           not be changed unless the client disconnects and reconnects.
```

Comment:
Should this block (with no title) be removed since we already talk about it in the first block ?

Reply:
I don't think so. This is about deprecation of the flag specifically in relation to the intercept command. There's no mention of that in the first block, which talks about the connect command.